### ISSUE TITLE
Fast scroller crashes from Fabric.io

### ISSUE THREAD

**Issue by afollestad** [afollestad/cabinet-issue-tracker#179]
https://fabric.io/aidan-follestads-projects/android/apps/com.afollestad.cabinet/issues/5534e10f5141dcfd8f847269

```groovy
java.lang.RuntimeException: Unable to start activity ComponentInfo{com.afollestad.cabinet/com.afollestad.cabinet.ui.MainActivity}: android.view.InflateException: Binary XML file line #20: Error inflating class xyz.danoz.recyclerviewfastscroller.FastScroller
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2334)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2392)
       at android.app.ActivityThread.access$900(ActivityThread.java:169)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1280)
       at android.os.Handler.dispatchMessage(Handler.java:102)
       at android.os.Looper.loop(Looper.java:146)
       at android.app.ActivityThread.main(ActivityThread.java:5487)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:515)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1283)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1099)
       at dalvik.system.NativeStart.main(NativeStart.java)
Caused by: android.view.InflateException: Binary XML file line #20: Error inflating class xyz.danoz.recyclerviewfastscroller.FastScroller
       at android.view.LayoutInflater.createView(LayoutInflater.java:626)
       at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:702)
```

https://fabric.io/aidan-follestads-projects/android/apps/com.afollestad.cabinet/issues/5535b3b65141dcfd8f865568

```groovy
java.lang.RuntimeException: Unable to start activity ComponentInfo{com.afollestad.cabinet/com.afollestad.cabinet.ui.MainActivity}: android.view.InflateException: Binary XML file line #20: Error inflating class xyz.danoz.recyclerviewfastscroller.FastScroller
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2062)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2087)
       at android.app.ActivityThread.access$600(ActivityThread.java:133)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1198)
       at android.os.Handler.dispatchMessage(Handler.java:99)
       at android.os.Looper.loop(Looper.java:146)
       at android.app.ActivityThread.main(ActivityThread.java:4777)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:511)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:789)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:556)
       at dalvik.system.NativeStart.main(NativeStart.java)
Caused by: android.view.InflateException: Binary XML file line #20: Error inflating class xyz.danoz.recyclerviewfastscroller.FastScroller
       at android.view.LayoutInflater.createView(LayoutInflater.java:613)
       at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:687)
```

https://fabric.io/aidan-follestads-projects/android/apps/com.afollestad.cabinet/issues/5535b3b65141dcfd8f865568

```groovy
java.lang.RuntimeException: Unable to start activity ComponentInfo{com.afollestad.cabinet/com.afollestad.cabinet.ui.MainActivity}: android.view.InflateException: Binary XML file line #20: Error inflating class xyz.danoz.recyclerviewfastscroller.FastScroller
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2062)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2087)
       at android.app.ActivityThread.access$600(ActivityThread.java:133)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1198)
       at android.os.Handler.dispatchMessage(Handler.java:99)
       at android.os.Looper.loop(Looper.java:146)
       at android.app.ActivityThread.main(ActivityThread.java:4777)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:511)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:789)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:556)
       at dalvik.system.NativeStart.main(NativeStart.java)
Caused by: android.view.InflateException: Binary XML file line #20: Error inflating class xyz.danoz.recyclerviewfastscroller.FastScroller
       at android.view.LayoutInflater.createView(LayoutInflater.java:613)
       at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:687)
```

**Comment by afollestad** [comment]
Any idea what would cause this @plusCubed?

**Comment by plusCubed** [comment]
@afollestad Attempt to fix it in afollestad/cabinet@fb350712bf57bf54a01f18db0b4f024fcbdde4f6. I don't know why a null background would cause problems, but let's see if this does the trick.

**Comment by afollestad** [comment]
Cool, thanks, we'll see

**Comment by afollestad** [comment]
It appears it's still there lol:

https://fabric.io/aidan-follestads-projects/android/apps/com.afollestad.cabinet/issues/5534e10f5141dcfd8f847269

Another issue showed up too, very weird:

https://fabric.io/aidan-follestads-projects/android/apps/com.afollestad.cabinet/issues/5534c7325141dcfd8f8439fc

**Comment by afollestad** [comment]
It also appears it happened to one user with the bread crumbs :o

https://fabric.io/aidan-follestads-projects/android/apps/com.afollestad.cabinet/issues/5533f4255141dcfd8f82d344

**Comment by afollestad** [comment]
Updated the original comment in this issue to show the current issues and their stack traces.

**Comment by afollestad** [comment]
These should hopefully be fixed now with the latest commits.

**Comment by afollestad** [comment]
Re-opening cause they're still there unfortunately

**Comment by plusCubed** [comment]
Found the cause (dynamic attributes on <5.0). Should be fixed in afollestad/cabinet@fdb261927c9df5b2c86088d1bafab1b7770ebdca.

**Comment by afollestad** [comment]
Sweet, thanks!
